dali/pipeline/pipeline_test.cc

```diff
@@ -446,7 +446,7 @@ TEST_F(PipelineTestOnce, TestPresize) {
   CUDA_CALL(cudaMemcpy(&tmp, ws.Output<GPUBackend>(2).mutable_data<size_t>(),
             sizeof(size_t) * 2, cudaMemcpyDefault));
   ASSERT_EQ(tmp[0], presize_val_Mixed);
-  ASSERT_EQ(tmp[1], 2 * sizeof(size_t));
+  ASSERT_EQ(tmp[1], Buffer<CPUBackend>::padding());
```

Comment:
It should be padded 2 in case of padding is changed to 0.

Reply:
Do you want something like:
`std::max(2 * sizeof(size_t), Buffer<CPUBackend>::padding())` here, because I don't exactly get this comment?

If so, I don't think that is that big of an issue, because the test will catch the changing behaviour of padding and require to fix it.

Reply:
Yes. It should do

Reply:
Done